grai-server/app/connections/task_helpers.py

```diff
@@ -71,11 +71,11 @@ def schemaToModel(item: T):
         values = item.spec.__dict__
 
         values["workspace"] = workspace
+        values["display_name"] = values["name"]
```

Comment:
Both Node and Edge should have a display name populated for them by default in their respective model `on_save` but you must have run into an issue here, what was the bug?

Reply:
I will try removing this line and see what happens, I think it might be a peculiarity of the way I am testing this code.

Reply:
I am not sure what is going on exactly here but the test fails without this line.